src/components/drag-and-drop/draggable-context/draggable-context.js

```diff
@@ -146,7 +148,12 @@ class DraggableContext extends React.Component {
    * Renders the component
    */
   render() {
-    return this.props.children;
+    return (
+      <div className='carbon-draggable-context'>
+        { this.props.children }
+        <CustomDragLayer />
```

Comment:
Yeah I like the idea of being able to pass in your own drag layer into the Draggable Context.